types/parse/index.d.ts

```diff
@@ -1,6 +1,8 @@
-// Type definitions for parse 1.9
+// Type definitions for parse 2.2.20
```

Comment:
I think the last number is important since the feature was added in release 2.2.20 and was not available in 2.2.19.
https://github.com/parse-community/parse-server/blob/master/CHANGELOG.md